src/components/get-involved-components/GetInvolvedCard.js

```diff
@@ -9,14 +9,22 @@ const GetInvolvedCard = ({ data }) => {
   const buttonText = data?.linkTitle;
   const link = data?.link;
 
+  const overlayStyle = {
+    backgroundSize: '100% 600%',
+    backgroundPosition: isHovering ? '100% 100%' : '100% 0%',
+    backgroundImage:
+      'linear-gradient(135deg, rgb(255,0,0,0), rgb(255,0,0,0), rgb(255,0,0,0), rgb(255,0,0,0.6))',
+  };
+
   return (
     <div
       onMouseEnter={() => setIsHovering(true)}
       onMouseLeave={() => setIsHovering(false)}
       className="relative flex flex-col w-10/12 transition-all duration-30 text-white 
```

Comment:
little typo here (duration-30), I know that wasn't you, but could we change that to 300

Reply:
done ✅